source/security/private-endpoints.txt

```diff
@@ -372,6 +372,15 @@ You can toggle this setting in three ways:
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
 
+IP Access Lists and Private Endpoints
+-------------------------------------
+
+When you enable private endpoints, you can still enable access to your App Services application using other methods, such as adding public IPs to :ref:`IP access lists 
```

Comment:
Nit - this line should be wrapped as it looks significantly longer than our 80 character line width.

Reply:
Ah, yep. Thank you! I committed one of Melodee's suggestions and didn't think the break the line.